pyccel/codegen/printing/cwrappercode.py

```diff
@@ -28,7 +30,8 @@ def get_new_name(self, used_names, requested_name):
             used_names.add(requested_name)
             return requested_name
         else:
-            return create_incremented_string(used_names, prefix=requested_name)
+            return 'random_name'
+       #     return create_incremented_string(used_names, prefix=requested_name)
```

Comment:
It is important to use the incremented string function as this prevents name collisions (it is not random)

Reply:
yes I know I didn't know to create that function so I used random name for the time being

Reply:
The function already exists it just needs importing from ast.core if it hasn't been yet